crates/e2e/tests/e2e/local_node.rs

```diff
@@ -103,3 +103,23 @@ impl<T: Transport> EvmApi<T> {
         CallFuture::new(self.transport.execute("evm_revert", vec![value_id]))
     }
 }
+
+pub struct AccountAssigner {
+    pub default_deployer: Account,
```

Comment:
Looks like `default_deployer` is only ever initialized but never read. Should that be?

Reply:
I intentionally wanted to exclude it from the available accounts since it's used under the hood by many things (default tx executor, default minter). I could just have skipped an account and added a comment, but I thought it was useful to keep this account easily available in case someone needs it. If you prefer I can remove it.

Reply:
That explanation seems reasonable. Was just worried that it might point to an error that the field itself was not used.